src/mapping/axioms.obo

```diff
@@ -1,2793 +0,0 @@
-format-version: 1.2
-ontology: rad/axioms
-
-[Term]
```

Comment:
hmm, a chemical entity should never be equivalent to an exposure to a chemical entity. We should filter these

Reply:
Does that need to be addressed before this PR can be merged?